prime-dotnet-webapi/Controllers/SigningAuthoritiesExtendedController.cs

```diff
@@ -0,0 +1,41 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+using Prime.Configuration.Auth;
+using Prime.Services;
+
+namespace Prime.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/parties/signing-authorities")]
+    [ApiController]
+    public class SigningAuthoritiesExtendedController : PrimeControllerBase
```

Comment:
I'm not sure the reason for the new controller?  If it has to do with `Authorize` attribute, an example with lots of different `Authorize` configurations is `ProvisionerAccessController`

Reply:
The existing SigningAuthoritiesController controls the access at the class level, unlike ProvisionerAccessController which at each action/function level. Instead of updating SigningAuthoritiesController `Authorize` to action/function level, adding a new one is better as we could remove the class all-together once it is no longer needed.

Reply:
I think the `Authorize` can be overridden at the action/function-level ... see `EnrolleesExtendedController` e.g.

Reply:
hm.. I just tried to put the new function to the `SigningAuthoriesController`, without changing the class level `Authorize` attribute, and the API call got 403 error. Then I removed the class `Authorize` attribute and the call went through. It seems the class level `Authorize` attribute is the base which applied to all functions. The function level ones are the additional rules. looking at the `EnrolleeExtendedController` `Authorize` attribute, the rule doesn't make much sense. Let me dig a bit deeper...

Reply:
this explains all:
![image](https://user-images.githubusercontent.com/108699279/228033653-db5a0f0c-df48-4373-9b21-91207f19aff0.png)

Reply:
Anyway, the current implementation is the best suit for this case in my opinion.